Userland/DevTools/UserspaceEmulator/SoftFPU.h

```diff
@@ -0,0 +1,665 @@
+/*
+ * Copyright (c) 2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2021, Leon Albrecht <leon2002.la@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Report.h"
+#include <LibX86/Instruction.h>
+#include <LibX86/Interpreter.h>
+
+#include <math.h>
+#include <string.h>
+
+namespace UserspaceEmulator {
+
+class Emulator;
+class SoftCPU;
+
+class SoftFPU final {
```

Comment:
Almost everything in this class is `fpu_` prefixed. You are already defining the methods in the `SoftFPU` class, you do not need to prefix them.

Reply:
this is to differentiate the x87 part and the mmx part of this class, so I see your point, but there is a reason for it though